tests/unit/test_arithmetic_expression.cpp

```diff
@@ -1692,7 +1692,7 @@ TEST_F(ArithmeticTest, ReduceTest) {
 
 	AR_ExpNode *lhs = arExp->op.children[0];
 	ASSERT_EQ(AR_EXP_OP, lhs->type);
-	ASSERT_EQ(0, strcasecmp("rand", lhs->op.func_name));
+	ASSERT_EQ(0, strcasecmp("rand", lhs->op.f->name));
```

Comment:
there should be a function which get's you the name of the function
this is digging deep into the structure destroying opacity